examples/ale_classification.ipynb

```diff
@@ -0,0 +1,787 @@
+{
```

Comment:
<ul><li>vice versa for the other two classes -&gt; this is inaccurate as the effect is anything but large pos/neg for the vericolor cls</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>